### ISSUE TITLE
Rama 'tell-me-my-age' apartado 1

### ISSUE THREAD

**Issue by JuanMaRuizH** [DevAppWeb/tell-me-my-age#1]
Buenas, por validar fechas se entiende que se debe comprobar que los días del mes se corresponden con dicho mes. Es decir, que se debe validar que el 31 de abril es una fecha incorrecta ¿cierto?

Gracias. Un saludo

**Comment by jivanrod** [comment]
Sí, así es, la fecha tiene que corresponder a una fecha real. También hay que tener en cuenta los años bisiestos.

Un saludo

**Comment by JuanMaRuizH** [comment]
Perfecto, muchas gracias
